hydragnn/utils/distributed/distributed.py

```diff
@@ -49,7 +49,7 @@ def find_ifname(myaddr):
     return ifname
 
 
-def parse_nodelist(nodelist):
+def parse_slurm_nodelist(nodelist):
```

Comment:
@jychoi-hpc 

You renamed `parse_nodelist` into `parse_slurm_nodelist` 

However, Aurora uses PBS, not SLURM.

Why did you rename the function by adding "slurm" in its name?